src/cli/onefuzz/status/cache.py

```diff
@@ -311,63 +345,60 @@ def should_render_job(self, job: Job) -> bool:
 
         return True
 
+    def job_stopped(self, event: EventJobStopped) -> None:
+        if event.job_id in self.jobs:
+            del self.jobs[event.job_id]
+
     def render_jobs(self) -> List[Tuple]:
         results: List[Tuple] = []
 
-        for (timestamp, job) in sorted(self.jobs.values(), key=lambda x: x[0]):
```

Comment:
Why do we no longer sort by timestamp?

Reply:
Once I had a 3k concurrent tasks in the view, sorting on timestamp caused the data to jump around far too frequently to be useful.